README.md

```diff
@@ -65,14 +66,10 @@ $ skopeo delete docker://localhost:5000/imagename:latest
 
 Private registries with authentication
 -
-When interacting with private registries, `skopeo` first looks for the Docker's cli config file (usually located at `$HOME/.docker/config.json`) to get the credentials needed to authenticate. When the file isn't available it falls back looking for `--username` and `--password` flags. The ultimate fallback, as Docker does, is to provide an empty authentication when interacting with those registries.
+When interacting with private registries, `skopeo` first looks for the Docker's cli config file (usually located at `$HOME/.docker/config.json`) to get the credentials needed to authenticate. When the file isn't available it falls back looking for `--creds` (for `skopeo inspect`) or `--src-creds|--dest-creds` (for `skopeo copy`) flags. The ultimate fallback, as Docker does, is to provide an empty authentication when interacting with those registries.
```

Comment:
I think this is wrong.  If the user specified --creds, --src-creds, --dest-creds, it should override content in config.json rather then fall back.